### PR TITLE
Translated backups.md into German

### PR DESCRIPTION
**Pull request by hfgd123**
## Type of PR

- [ X ] Translation update / creation
- [ ] Original file creation
- [ ] Original file update

## Checks

If you have changed an original file, please confirm:

- [ ] Yes, I have copied the updated section to all translations of this file manually.

## Notes

- Did not translate specific error messages in the Troubleshooting part as I don't have the exact wording in the German version.
- Did not translate "THIS FEATURE MIGHT NOT BE ABLE TO EVERYONE" as I don't know what that means

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+sidebar_position: 2
+---
+
+# Server Backups
+
+Sichere die Nachrichten, Kanäle, Rollen und mehr deines Server automatisch und stelle es wieder her sobald irgendetwas mal schiefgehen sollte.
+
+:::tip Deine Privatsphäre ist uns wichtig
+Da Backups vertrauliche Nutzerinformationen und Nachrichten beinhalten werden sie verschlüsselt gespeichert und nur während sie aktiv verwendet werden (z.B. während der Wiederherstellung) entschlüsselt. Trotzdem weisen wir dich darauf hin, möglichst keine Nachrichten die private Daten enthalten (wie Kreditkartendaten, Sozialversicherungsnummern oder ähnliches) in Kanäle die in Backups enthalten sind zu senden.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:10]:
```suggestion
Da Backups vertrauliche Nutzerinformationen und Nachrichten beinhalten, werden sie verschlüsselt gespeichert und nur während sie aktiv verwendet werden (z.B. während der Wiederherstellung) entschlüsselt. Trotzdem weisen wir dich darauf hin, möglichst keine Nachrichten, die private Daten enthalten (wie Kreditkartendaten, Sozialversicherungsnummern oder ähnliches), in Kanäle, die in Backups enthalten sind, zu senden.
```
Please respect the German construct of "Relativsatz"

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Kanäle              | Backups werden jeden Kanal auf deinem Discord enthalten. Dies beinhalten: Kanalname, Kanalbeschreibung, Cooldown, Kanalberechtigungen und mehr |
+| Rollen              | Backups werden jede Rolle auf deinem Discord enthalten. Gespeicherte Rollen-Daten beinhalten: Berechtigungen, Name, Farbe und mehr                                       |
+| Emojis              | Backups werden jedes Emoji auf deinem Discord enthalten. Dies Beinhaltet das Emoji-Bild an sich sowie den emoji-name. Sticker sind ggf. auch mit inbegriffen.                |
+| Banns               | Backups werden jeden Bann auf deinem Discord enthalten. Jeder Bann beinhaltet den Grund und den gebannten Nutzer.                                                       |
+| Mitglieder          | Backups werden beinhalten welcher Nutzer welche Rolle zum Zeitpunkt der Backup-Erstellung hatte.<br/> **THIS FEATURE MIGHT NOT BE ABLE TO EVERYONE**                        |
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:21]:
```suggestion
| Mitglieder          | Backups werden beinhalten welcher Nutzer welche Rolle zum Zeitpunkt der Backup-Erstellung hatte.<br/> **DIESE FUNKTION IST EVENTUELL NICHT FÜR ALLE VERFÜGBAR**                        |
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+## Manuelle Backups {#manual}
+
+<IncludedInPlan data={{PRO: true, ACTIVE_GUILD: true, STARTER: true}} additionalDetails={{
+    STARTER: 'Speicherung von bis zu 2 Backups gleichzeitig, jedes mit bis zu 10 Nachrichten pro Kanal',
+    PRO: 'Speicherung von bis zu 10 Backups gleichzeitig, jedes mit bis zu 50 Nachrichten pro Kanal',
+    ACTIVE_GUILD: 'Speicherung von bis zu 5 Backups gleichzeitig, jedes mit bis zu 20 Nachrichten pro Kanal'
+}}/>
+
+Um manuell Backups zu erstellen, öffne dein [Backup-Dashboard](https://scnx.app/de/glink?page=backups) im SCNX Dashboard. Klicke danach auf "Backup erstellen" und wähle was [in deinem Backup enthalten sein soll](#included). Bestätige danach nocheinmal dein Backup.
+Ein Backup zu erstellen kann bis zu 15 Minuten dauern. Dies ist abhängig von der Anzahl der zu speichernden Nachrichten.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:33]:
```suggestion
Ein Backup zu Erstellen kann bis zu 15 Minuten dauern. Dies ist abhängig von der Anzahl der zu speichernden Nachrichten.
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+![](@site/docs/assets/scnx/guilds/backups/create.png)
+
+Nach der Erstellung des Backups (dies kann eine Weile dauern) kannst du es in der Backup-Liste ansehen und verwalten. Folgende Optionen sind verfügbar:
+* Backup wiederherstellen: Erm,öglicht dir das Backup auf diesem oder einem anderen Server wiederherzustellen. Im Abschnitt [Backups wiederherstellen](#restore) kannst du mehr darüber erfahren.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:38]:
```suggestion
* Backup wiederherstellen: Ermöglicht dir das Backup auf diesem oder einem anderen Server wiederherzustellen. Im Abschnitt [Backups wiederherstellen](#restore) kannst du mehr darüber erfahren.
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+<IncludedInPlan data={{PRO: true, ACTIVE_GUILD: true, STARTER: true}}/>
+
+### Wiederherstellung auf dem ursprünglichen Server {#restore-self}
+
+Bevor du ein Backup wiederherstellst, passe bitte folgende Einstellungen an deinem Discord an:
+* Schiebe die Rolle des (verifizierten) SCNX Bots über alle anderen Rollen und gib ihr Administrator-Berechtigungen
+* Deaktiviere die "Community"-Einstellung deines Servers in den Einstellungen (du kannst sie danach wieder aktivieren)
+* Verstehe, dass dieser Vorgang alle Nachrichten, Kanäle und Rollen von deinem Server löscht
+
+Um ein Backup auf deinem Server wiederherzustellen, öffne das [Backup-Dashboard](https://scnx.app/de/glink?page=backups) deines Servers und klicke auf das Backup, das du wiederherstellen möchtest. Als nächstes klicke auf "Backup wiederherstellen". Dir wird ein Befehl angezeigt werden. Kopiere ihn mit dem "Befehl kopieren" Button und füge ihn in deinem Server ein. Stelle sicher, dass du den Befehl mit dem (verifizierten) SCNX-Bot ausführst.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:70]:
```suggestion
Um ein Backup auf deinem Server wiederherzustellen, öffne das [Backup-Dashboard](https://scnx.app/de/glink?page=backups) deines Servers und klicke auf das Backup, das du wiederherstellen möchtest. Als nächstes klicke auf "Backup wiederherstellen". Dir wird ein Befehl angezeigt werden. Kopiere ihn mit dem "Befehl kopieren" Knopf und füge ihn in deinem Server ein. Stelle sicher, dass du den Befehl mit dem (verifizierten) SCNX-Bot ausführst.
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+![](@site/docs/assets/scnx/guilds/backups/execute-command.png)
+
+Bestätige als Nächstes deine Anfrage mit dem Button unter der Antwort in Discord.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:74]:
```suggestion
Bestätige als Nächstes deine Anfrage mit dem Knopf unter der Antwort in Discord.
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+### Wiederherstellung auf anderen Servern {#restore-other-server}
+
+:::tip
+Der Server auf dem du das Backup wiederherstellen möchtest muss nicht zu SCNX hinzugefügt sein, aber der [SCNX Bot muss eingeladen werden](https://sc-net.work/invite-scnx).
+:::
+
+Der Wiederherstellungsprozess ist größtenteils identisch zur [Wiederherstellung auf dem ursprünglichen Server](#restore-self) - selbst der verwendete Befehl bleibt gleich. Du musst allerdings die Server-Sperre auf deinem Server deaktivieren.
+
+Die Server-Sperre ist ein Sicherheitsmechanismus für dein Backup: Wenn sie deaktiviert ist, kann jeder mit dem Backup-Code (der Befehl der im Dashboard angezeigt wird) das Backup auf jedem Server wiederherstellen. Zum Glück ist die Sperre standardmäßig aktiviert. Um sie zu deaktivieren,öffne das [Backup-Dashboard](https://scnx.app/de/glink?page=backups), wähle dein Backup und klicke auf "Server-Sperre deaktivieren".
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:92]:
```suggestion
Die Server-Sperre ist ein Sicherheitsmechanismus für dein Backup: Wenn sie deaktiviert ist, kann jeder mit dem Backup-Code (der Befehl der im Dashboard angezeigt wird) das Backup auf jedem Server wiederherstellen. Zum Glück ist die Sperre standardmäßig aktiviert. Um sie zu deaktivieren, öffne das [Backup-Dashboard](https://scnx.app/de/glink?page=backups), wähle dein Backup und klicke auf "Server-Sperre deaktivieren".
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+    <li>Stelle sicher, dass du die "Community"-Einstellung deaktiviert hast und versuche es erneut.</li>
+    <li>Stelle sicher, dass die Rolle des (verifizierte) SCNX Bots über allen anderen Rollen steht und versuche es erneut</li>
+    <li>Kontaktiere bei wiederholtem Auftreten [unser Team](https://scnx.app/de/help). Dieses wird dich bei der Problemsuche unterstützen.</li>
+</details>
+<details>
+    <summary>
+        SCNX erstellt keine automatischen Backups auf deinem Server
+    </summary>
+    <li>Stelle sicher, dass der (verifizierte) SCNX Bot die Administrator Berechtigung hat und versuche es erneut.</li>
+    <li>Stelle sicher, dasss du [diese Funktion](#automatic] wirklich aktiviert hast und alle Änderungen gespeichert hast.</li>
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:135]:
```suggestion
    <li>Stelle sicher, dasss du [diese Funktion](#automatic) wirklich aktiviert hast und alle Änderungen gespeichert hast.</li>
```

**Review by SCDerox — CHANGES_REQUESTED** [review]
Thanks for your contributions. I've adjusted a few things, please review them.

 I belive that none of the error messages are localised into German atm.

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md (hunk 6 above)
Comment (hfgd123) [reply i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:74]: For virtual buttons "Knopf" somehow sounds weird... But I guess it is correct language-wise

**Review by SCDerox — APPROVED** [review]
lgtm

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md (hunk 6 above)
Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md]: I find "Schaltfläche" even more weird then "Knopf", so what should we use?

Comment (hfgd123) [reply i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md]: Idk. German doesn't really have a good word considering context and targeted audience here

Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md]: I agree, that's why I choose "Knopf", as this is the most intuitive translation of "Button" and describes the function of a button most

**Pull request merged**